cmd/veneur-proxy/main.go

```diff
@@ -23,6 +23,7 @@ import (
 
 var (
 	configFile = flag.String("f", "", "The config file to read for settings.")
+	strict     = flag.Bool("strict", false, "Fails if unknown config fields are present")
```

Comment:
what are the cases where we wouldn't want to always turn it on?

Reply:
An example would be a config migration where there will temporarily be an old field.